docs/source/conf.py

```diff
@@ -260,9 +260,6 @@
 redirects = {
     "intro/zen": "https://asreview.ai/blog/the-zen-of-elas/",
     "intro/faq": "https://github.com/asreview/asreview/discussions",
-    "intro/about.html#research-team": "https://asreview.nl/about/",
-    "intro/about.html#the-case-of-systematic-reviewing": "https://asreview.nl/blog/asreview-class-101/",
```

Comment:
@Rensvandeschoot Tested. These kinds of redirects won't work. Visiting `intro/about.html#nonexisting` just lands on `intro/about`.

Reply:
too bad! suggestions?